ddr/tools/CMakeLists.txt

```diff
@@ -21,3 +21,5 @@
 
 add_subdirectory(blob_reader)
 add_subdirectory(ddrgen)
+
+export(TARGETS omr_blob_reader omr_ddrgen FILE "DDRTools.cmake")
```

Comment:
Where does `DDRTools.cmake` come from?

Reply:
This statement generates the `DDRTools.cmake` file

Reply:
I still don't get it; `export` declares that this project produces `DDRTools.cmake` - my question is how?

Reply:
https://cmake.org/cmake/help/v3.12/command/export.html

>Export targets from the build tree for use by outside projects. 
>Create a file `filename` that may be included by outside projects to import targets from the current project’s build tree.

Reply:
Let me rephrase: what defines the content of that file?

Reply:
its internally defined by cmake. When you include it it will define an imported target with the appropriate paths, etc. Here is a sample from when we export hookgen/tracegen
https://gist.github.com/dnakamura/24ecce3061474162e890f72838551619

Reply:
Certainly cmake doesn't know anything about hookgen or tracegen: where is the information that tells it what to put in that file.

Reply:
https://github.com/eclipse/omr/blob/ded9f8c63f3ca7070eb7ee16f89612445ede4f8f/tools/CMakeLists.txt#L38

i believe it just puts the location where the targets are built